packages/phone-number-privacy/.vscode/launch.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
what is this for?  should the .vscode path be added to .gitignore?

Reply:
This file allows us to debug tests locally in VSCode. I figured others would find it useful but we can always have folks generate it on-demand. Do you have any preference?

Reply:
i don't have a preference -- i typically see the .vscode stuff added to .gitignore but am not an expert on the best practices here

more from a curiosity, how is it used?

Reply:
VSCode expects the .vscode to be at the root of the directory that is currently open in VSCode. However, we can add multiple Launch.json files at the root of monorepo and the dev can select whichever one she wants to run: https://code.visualstudio.com/Docs/editor/debugging

Reply:
I'll add this as a separate PR and get broader eyes on it since it'll be setting a precedence across the repo.